api/global/internal/meter.go

```diff
@@ -233,6 +233,9 @@ func (bound *instHandle) RecordOne(ctx context.Context, number core.Number) {
 	if implPtr == nil {
 		implPtr = (*metric.BoundInstrumentImpl)(atomic.LoadPointer(&bound.delegate))
 	}
+	if implPtr == nil {
+		return
```

Comment:
this needs comments explaining why/how this could occur, similar to what you've put in the PR description.

Reply:
Done.